carbon-design-system/carbon
# fix(storybook): update each folder to have a "Default" story

This updates every storybook component folder to have a story named "Default" where possible. This is in support of the platform beginning to consume our stories.

#### Changelog

**Changed**

- update any component story file where ComponentName/ComponentName instead to be ComponentName/Default

#### Testing / Reviewing

For each component modified:
- Ensure the Default story renders
- Ensure there are no regressions on the docs tab
